styleguide/_themes/derek/scss/snowflakes/filtering.scss

```diff
@@ -0,0 +1,94 @@
+.rsFilter-section {
+  overflow: hidden;
+}
+
+.rsFilter-bar {
+  @include make-sm-column(2);
+  background-color: $gray-lighter;
+  padding: 20px;
+}
+
+.rsFilter-hamburger {
+  background-color: transparent;
+  border: 0;
+  display: none;
+  padding: 8px 12px;
+
+  &::before {
+    content: '\f0c9';
+    font-family: 'FontAwesome';
+  }
+
+  &:hover {
+    background-color: $gray-light;
+  }
+
+  &:focus {
+    outline: 0;
+  }
+}
+
+.rsFilter-title-mobile {
+  @include subhead($gray-darker);
+  display: none;
+  text-align: center;
+}
+
+.rsFilter-form-title {
+  @include subhead($gray-darker);
+  text-align: center;
+}
+
+.rsFilter-form-select {
+  background: $white;
+  display: block;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  width: $full-width;
+}
+
+.rsFilter-button {
+  background-color: $gray-base;
+  border: 0;
+  color: $white;
+  display: block;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  padding: 0 15px;
+  text-align: center;
+  width: $full-width;
+
+  &:hover {
+    background-color: $gray-darker;
+    cursor: pointer;
+  }
+
+  &:focus {
+    outline: 0;
+  }
+}
+
+.rsFilter-content {
+  @include make-sm-column(10);
+  padding: 25px;
+}
+
+// Responsive behavior
+@media (max-width: $screen-sm-min) {
```

Comment:
Check this breakpoint.  max-widths should technically match `*-max` variables (although this isn't always the case), but general rule of thumb here.  Could be related to the break point issues.